src/storage/Selector.ts

```diff
@@ -172,11 +191,19 @@ export class Selector <T> {
   }
 
   combine(... selectMetas: Selector<T>[]): Selector<T> {
-    const isEqual = selectMetas.every(meta => meta.select === this.select)
-    if (!isEqual) {
-      throw TOKEN_INVALID_ERR()
+    return Selector.combineFactory(this, ... selectMetas)
+  }
+
+  concat(... selectMetas: Selector<T>[]): Selector<T> {
+    const equal = selectMetas.every(m =>
```

Comment:
consider
```
SELECT * FROM TABLE WHERE id >=0 SKIP 1 LIMIT 10
SELECT * FROM TABLE WHERE id >=0 SKIP 2 LIMIT 20
...
```

these query will violate constraint of `concat` method, maybe we should add a term to check if `skip mod limit` is equal to `0`